### PR TITLE
[CCI] Fix bottom bar visibility using createPortal

### PR DESCRIPTION
**Pull request by SergeyMyssak**
### Description

Fixed bottom bar visibility using createPortal.

https://user-images.githubusercontent.com/17855243/236097081-de85528f-d784-47c3-9e0c-5f5965c44181.mov

To clarify, the `<EuiPortal />` component cannot be used as a replacement for `createPortal` function. This is because `<EuiPortal />` creates a `<div>` wrapper element that breaks the "sticky" behavior.

Although there have been some efforts to make `<EuiPortal />` work in a similar way to `createPortal` function, more information about this can be found in https://github.com/opensearch-project/OpenSearch-Dashboards/pull/3949 

### Issues Resolved

https://github.com/opensearch-project/OpenSearch-Dashboards/issues/3336

### Check List

- [x] All tests pass
  - [x] `yarn test:jest`
  - [x] `yarn test:jest_integration`
  - [x] `yarn test:ftr`
- [ ] New functionality includes testing.
- [ ] New functionality has been documented.
- [x] Update [CHANGELOG.md](./../CHANGELOG.md)
- [x] Commits are signed per the DCO using --signoff

### CODE REVIEW

File: src/core/public/rendering/_base.scss [hunk 1]
```diff
@@ -5,7 +5,6 @@
  */
 // SASSTODO: Naming here is too embedded and high up that changing them could cause major breaks
 #opensearch-dashboards-body {
-  overflow-x: hidden;
```
Comment (ashwin-pc) [inline src/core/public/rendering/_base.scss:8]: Why is this change necessary?

**Review by ashwin-pc — APPROVED** [review]
Nice! This is a much better solution. Great job on digging into this issue. Also kudos on figuring out the problem using EuiPortal here. This will inform the decision to modify the BottomBar component in OUI.

File: src/core/public/rendering/_base.scss (hunk 1 above)
Comment (SergeyMyssak) [reply src/core/public/rendering/_base.scss:8]: It's a known problem that sticky behavior doesn't work if we use `overflow: hidden` on the parent component. More information can be found at https://github.com/w3c/csswg-drafts/issues/865. I don't really know why this property was written, but it seems like it was just inherited from the old code on refactoring.

History of changes: 
1) https://github.com/opensearch-project/OpenSearch-Dashboards/commit/b5767804c131dfecb6f588584ca6f6801b503beb#diff-b735d2f2b9d87d3791ea90563e79a98455f186249f531b068f629b46adb5b24cR9
2) https://github.com/opensearch-project/OpenSearch-Dashboards/commit/1a99df72ba52581aef83d260d3ffc3238337b23e#diff-f2168606829459ec979e10448c792f49c49ca94dc0ef1eee8f4d7e3c63427445R10
3) https://github.com/opensearch-project/OpenSearch-Dashboards/commit/e6ca5faa9c9d34c4a9528466428813f603b22d45#diff-f2168606829459ec979e10448c792f49c49ca94dc0ef1eee8f4d7e3c63427445R6
4) https://github.com/opensearch-project/OpenSearch-Dashboards/commit/585a84cc9678bb14d1dccb66b40a06d57dfaaf77#diff-f2168606829459ec979e10448c792f49c49ca94dc0ef1eee8f4d7e3c63427445R5

Comment (ashwin-pc) [reply src/core/public/rendering/_base.scss:8]: Makes sense, Im good with that

**Review by ananzh — APPROVED** [review]
